src/pages/billing/models/AzureManagedAppCoordinates.ts

```diff
@@ -0,0 +1,7 @@
+export interface AzureManagedAppCoordinates {
+  tenantId: string // UUID as string
+  subscriptionId: string // UUID as string
+  managedResourceGroupId: string
+  region?: string
```

Comment:
Do we ever have a region?

Reply:
It's in the BPM's api docs. I don't know if we use it or not.
Same situation for `applicationDeploymentName`. 

We could remove them, with the idea that this is only the fields returned by both BPM _and_ rawls, then extend it later, if necessary.